CHANGELOG.md

```diff
@@ -7,15 +7,16 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+## Main (unreleased)
```

Comment:
Why did all headers for each release change?

Reply:
Fixed